app/_data/docs_nav_mesh_2.3.x.yml

```diff
@@ -1,680 +1,186 @@
-product: mesh
-release: 2.3.x
 generate: true
 assume_generated: true
-items:
-  - title: Introduction
-    icon: /assets/images/icons/documentation/icn-flag.svg
-    items:
-      - text: About service meshes
-        url: /introduction/what-is-a-service-mesh/
-        src: /.repos/kuma/app/_src/introduction/about-service-meshes
-      - text: Overview of Kong Mesh
-        url: /mesh/2.3.x/
-        absolute_url: true
-      - text: How Kong Mesh works
-        url: /introduction/how-kong-mesh-works/
-        src: /.repos/kuma/app/_src/introduction/how-kuma-works/
-      - text: Architecture
-        url: /introduction/architecture/
-        src: /.repos/kuma/app/_src/introduction/architecture/
-      - text: Stages of software availability
-        url: /availability-stages/
-      - text: Version support policy
-        url: /support-policy/
-      - text: Mesh Requirements
-        url: /introduction/mesh-requirements/
-        src: /.repos/kuma/app/_src/introduction/kuma-requirements/
-      - text: Release notes
-        url: /mesh/changelog
-        generate: false
-        absolute_url: true
-  - title: Getting Started
-    icon: /assets/images/icons/documentation/icn-quickstart-color.svg
-    url: /gettingstarted
-  - title: Kong Mesh in Production
-    icon: /assets/images/icons/documentation/icn-deployment-color.svg
-    items:
-      - text: Overview
-        url: /production/
-        src: /.repos/kuma/app/_src/production/
-      - text: Deployment topologies
-        items:
-          - text: Overview
-            url: /production/deployment/
-            src: /.repos/kuma/app/_src/production/deployment/
-          - text: Standalone deployment
-            url: /production/deployment/stand-alone/
-            src: /.repos/kuma/app/_src/production/deployment/stand-alone/
-          - text: Multi-zone deployment
-            url: /production/deployment/multi-zone/
-            src: /.repos/kuma/app/_src/production/deployment/multi-zone/
-          - text: Default ports
```

Comment:
This one is no longer being displayed

Reply:
Yes it's was inlined in the `kuma` docs a while back.